### PR TITLE
Use form encoding for all parameters in JWT

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

File: extensions/amp-access/0.1/amp-access-server-jwt.js [hunk 1]
```diff
@@ -266,6 +266,13 @@ export class AccessServerJwtAdapter {
         'state': this.serverState_,
         'jwt': encoded,
       };
+      let requestString = '';
+      for (const k in request) {
```
Comment (jridgewell) [inline extensions/amp-access/0.1/amp-access-server-jwt.js]: Do we not have a method for this already?

Comment (dvoytenko) [inline extensions/amp-access/0.1/amp-access-server-jwt.js]: No, but probably worth to do. I'll add.

Comment (dvoytenko) [inline extensions/amp-access/0.1/amp-access-server-jwt.js]: Done. PTAL

File: src/url.js [hunk 2]
```diff
@@ -164,12 +164,28 @@ export function addParamToUrl(url, key, value, opt_addToFront) {
...
+ * @return {string}
+ */
+export function serializeQueryString(params) {
+  let s = '';
+  for (const k in params) {
+    const v = params[k];
+    if (v == null) {
+      continue;
+    }
+    if (s.length > 0) {
```
Comment (jridgewell) [inline src/url.js]: Using an array might be a little cleaner:

```js
//...
s.push(`${encodeURIComponent(k)}=${encodeURIComponent(v)}`);
//...
return s.join('&');

**Comment by codewiz** [comment]
LGTM

File: src/url.js (hunk 2 above)
Comment (dvoytenko) [inline src/url.js]: Definitely. Done.

**Pull request merged**